test/helpers/application_helper_test.rb

```diff
@@ -224,4 +224,11 @@
       link_to_url("b").must_equal "<a href=\"b\">b</a>"
     end
   end
+
+  describe "#time_fomratting" do
```

Comment:
- this does not cover all code paths and breaks `ruby test/helpers/application_helper_test.rb` on master 
 - describe does not match the method under test (render_time)

addressing both issues in https://github.com/zendesk/samson/pull/858